libplatsupport/CMakeLists.txt

```diff
@@ -134,7 +134,7 @@ if(KernelPlatImx8mq OR KernelPlatformImx8mm-evk)
     target_include_directories(platsupport PUBLIC plat_include/imx8m)
 endif()
 
-if(KernelPlatformZynqmpZcu102 OR KernelPlatformZynqmpUltra96)
+if(KernelPlatformZynqmpZcu102 OR KernelPlatformZynqmpUltra96 OR KernelPlatformZynqmpUltra96v2)
```

Comment:
We could also use `if(KernelPlatformZynqmp)` here, it would be a step towards unification.

Reply:
Let's leave that one for a separate PR.